lighthouse-core/audits/accessibility/image-alt.js

```diff
@@ -33,6 +33,9 @@ class ImageAlt extends AxeAudit {
       category: 'Accessibility',
       name: 'image-alt',
       description: 'Every image element has an alt attribute',
+      helpText: 'Screen readers can\'t read images to their users. The `alt` ' +
```

Comment:
Suggest changing to something like "Screen reader users rely on alt text to provide descriptions for images. It's also used as fallback content if the image fails to load"